docker-compose.yml

```diff
@@ -3,4 +3,4 @@ version: '3'
 services:
   basex:
     build: .
-    image: hgdata1/basex:9.2.4-hg-2019.09.18T12.00.00
+    image: hgdata1/basex:9.2.4-hg-2021.01.15T10.00.00
```

Comment:
After thinking about it, since this is a (public) fork of BaseX we could consider deleting this specific reference but I don't think it really matters.